extension/src/shared/ui-components/Label.tsx

```diff
@@ -0,0 +1,6 @@
+import styled from "@emotion/styled";
+
+export const Label = styled("div")(() => ({
```

Comment:
I think this should be placed inside of `ui-components/MyData/`.

Reply:
@keiya01 the label is used with feedback and My-data